delivery_readmes/DELIVERY.README.RNASeq.txt

```diff
@@ -0,0 +1,119 @@
+README
+=======
+
+The README describes the content of the delivered data.
+-----------------------------------------------------------------
+
+====================================================================
+ProjectID
+====================================================================
+The root folder, which is named by the NGI project ID, contains one report folder, one analysis results folder and a folder for each sample.
+Each sample folder is accompanied by a .lst-file containing a list of the files in the folder and a .md5-file containing the MD5-checksums of the files in the folder.
+Use the MD5-checksums to verify the integrity of the files after transfer.
+
+├──ProjectID
+	├── 00-Reports
+	├── 01-RNA-Results
+        ├── Sample1
+        ├── Sample1.lst
+        ├── Sample1.md5
+        ├── Sample2
+        ├── Sample2.lst
+        ├── Sample2.md5
+	...
+	...
+        ├── SampleN
+        ├── SampleN.lst
+        └── SampleN.md5
+
+
+The root folder contains following files as well:
+
+├──ProjectID
+        ├── ACKNOWLEDGEMENTS.txt
+        ├── DELIVERY.README.RNASeq.txt
+        ├── miscellaneous.lst
+        └── miscellaneous.md5
+
+--ACKNOWLEDGEMENTS.txt
+A file describing what to include in any publications using this data.
+
+--miscellaneous.lst
+A list of the files of the delivery, excluding raw data.
+
+--miscellaneous.md5
+A file to check the integrity of the miscellaneous.lst file.
+
+
+File integrity checks
+---------------------
+It is extremely important to check the integrity of the delivered data using the .md5 files. To do this, run the following:
+md5sum -c file_with_checksums.md5
+For more information, please see https://ngisweden.scilifelab.se/resources/data-delivery/
```

Comment:
Whitespace around commands, make it clearer that the command contains an example string that should be replaced.

```suggestion
It is extremely important to check the integrity of the delivered data using the .md5 files. To do this, run the following command in the terminal (replace with the relevant file name):

md5sum -c [checksum_file.md5]

For more information, please see https://ngisweden.scilifelab.se/resources/data-delivery/
```